src/sentry/snuba/sessions_v2.py

```diff
@@ -297,7 +297,7 @@ class InvalidParams(Exception):
 
 
 def get_constrained_date_range(
-    params, allow_minute_resolution=False, max_points=MAX_POINTS
+    params, allow_minute_resolution=False, isProject=False, max_points=MAX_POINTS
```

Comment:
we should try to add the new behavior without modifying `get_constrained_date_range`, or making the parameter more generalizable since this is intended to be a utility function used in other places.